### ISSUE TITLE
add spoiler option to submit page

### ISSUE THREAD

**Issue by plebeius-eth** [plebbit/seedit#191]
I think the user should be able to post something marking it as spoiler right away. Some kind of "mark as NSFW" button can also be added later.

**Comment by plebeius-eth** [comment]
e4a50feec7e6abb179daba9bddca63bc6516ddfc

**Comment by estebanabaroa** [comment]
why is it "mark as spoiler", seems redundant, it should just be [ ] spoiler

here's how they do it on new reddit
![image](https://github.com/plebbit/seedit/assets/90033261/aadf7e0c-e695-4511-8e28-f0786ac7693e)

**Comment by plebeius-eth** [comment]
I agree it was redundant, fixed 1b88937532fc221f42c355ca17cca9c43581360f

**Comment by estebanabaroa** [comment]
I think the spoiler should probably only apply to the media, not the text. I wrote about why here https://github.com/plebbit/plebchan/issues/368#issuecomment-2156803758

**Comment by plebeius-eth** [comment]
This issue was completed, it's only about making a comment as spoiler (`comment.spoiler`) which adds the SPOILER label to the comment, and blurs media if any.

![Image](https://github.com/user-attachments/assets/bd5ca72e-d8c5-457d-ba3f-7894bd4590d9)
